package.json

```diff
@@ -71,9 +71,9 @@
     "npm-run-all": "^4.0.2",
     "nyc": "^10.2.0",
     "rimraf": "^2.5.4",
-    "tslint": "^5.4.2",
+    "tslint": "^5.5.0",
     "tslint-test-config-non-relative": "file:test/external/tslint-test-config-non-relative",
-    "typescript": "2.4.1"
+    "typescript": "^2.4.1"
```

Comment:
This is probably not a great idea; we should use `~2.4.1` since typescript has breaking changes across minor versions. `~` forces us to be explicit about upgrades, which is good.